### PR TITLE
feat: [CDS-39773]: Re-Design Glance cards and updated icons for pipelines, services and infrastructures

### PR DESCRIPTION
**Pull request by Sagar-Rajput7**
#### Screenshots

Removed bg color from project card and fixed font and trend icon change

Earlier : 
image

Now: 
image

image

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by Sagar-Rajput7** [comment]
retrigger titlecheck

**Review by bhavya-sinha — DISMISSED** [review]

**Review by depanshu-dhiman — APPROVED** [review]

**Review by praneshg239 — APPROVED** [review]

**Review by avinashmadhwani02 — DISMISSED** [review]

**Review by vkbansal — APPROVED** [review]

**Review by praneshg239 — DISMISSED** [review]

**Review by praneshg239 — APPROVED** [review]

File: src/modules/45-projects-orgs/components/OverviewGlanceCards/OverviewGlanceCards.tsx [hunk 1]
```diff
@@ -74,13 +75,33 @@ const getDataForCard = (
...
   if (countChange) {
-    glanceCardData.delta = countChange > 0 ? `+${countChange.toString()}` : countChange.toString()
+    glanceCardData.intent = countChange > 0 ? 'success' : 'danger'
+    const rateColor = countChange > 0 ? 'var(--green-800)' : 'var(--red-700)'
+    glanceCardData.delta = (
+      <Layout.Horizontal>
+        <Icon
+          size={14}
+          name={countChange > 0 ? 'caret-up' : 'caret-down'}
+          style={{
```
Comment (depanshu-dhiman) [inline src/modules/45-projects-orgs/components/OverviewGlanceCards/OverviewGlanceCards.tsx:90]: Use classnames here. Try `!important` if style is not working from classname

File: src/modules/45-projects-orgs/components/OverviewGlanceCards/OverviewGlanceCards.tsx [hunk 2]
```diff
@@ -94,7 +115,7 @@ const RenderGlanceCard: React.FC<RenderGlanceCardProps> = props => {
       <Icon name="spinner" size={24} color={Color.PRIMARY_7} />
     </Card>
   ) : (
-    <GlanceCard {...data} styling={data.title === projectsTitleId} title={getString(data.title)} />
+    <GlanceCard {...data} styling={false} title={getString(data.title)} />
```
Comment (depanshu-dhiman) [inline src/modules/45-projects-orgs/components/OverviewGlanceCards/OverviewGlanceCards.tsx:118]: No need to pass false as it will default to false

**Review by depanshu-dhiman — CHANGES_REQUESTED** [review]
Minor comments

**Review by depanshu-dhiman — DISMISSED** [review]

File: src/modules/45-projects-orgs/components/OverviewGlanceCards/OverviewGlanceCards.tsx (hunk 1 above)
Comment (Sagar-Rajput7) [reply src/modules/45-projects-orgs/components/OverviewGlanceCards/OverviewGlanceCards.tsx:90]: will fix this is another pr that i am raising by EOD

File: src/modules/45-projects-orgs/components/OverviewGlanceCards/OverviewGlanceCards.tsx (hunk 2 above)
Comment (Sagar-Rajput7) [reply src/modules/45-projects-orgs/components/OverviewGlanceCards/OverviewGlanceCards.tsx:118]: will fix this is another pr that i am raising by EOD

**Pull request merged**